app/models/bottleneck_event.rb

```diff
@@ -47,6 +47,25 @@ def self.calculate_future_event(obj, options)
     send(method, obj, options)
   end
 
+  def self.build_event_message(context)
+    value =
+      case context[:event][:event_type]
+      when "CpuUsage"
+        MiqReport.new.format_mhz_to_human_size(context[:limit_attr_value], :precision => 1)
+      when "MemoryUsage"
+        MiqReport.new.format_bytes_to_human_size(context[:limit_attr_value].megabytes, :precision => 1)
+      when "DiskUsage"
+        MiqReport.new.format_bytes_to_human_size(context[:limit_attr_value], :precision => 1)
+      end
+
+    "%{trend_attr} is projected to reach %{limit_value} (%{limit_pct}%% of %{limit_attr})" % {
```

Comment:
I was tempted to make this translated, since I think it could get presented to the end user. Thoughts?  It's easy enough to just wrap in `_()`

Reply:
Not sure what you mean - the UI does this the same way with the dictionary?

Reply:
ok - found it - I was thinking all the stuff around `miq_report_formats.yml` - it was wrong.

As long as the output of this message is the ui and not in the db, lets translate it.

Reply:
Sorry, I meant to comment on this earlier but didn't hit the button.  I think it's worth translating but I wonder if the phrasing makes it harder to translate correctly.  Maybe we can reorder the sentence or break it into two phrases that can be translated separately.  Not sure.  I think we can do it for followup.